### ISSUE TITLE
make tables more accessible

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/hourglass#410]
from work on Janeway
- openlibhums/janeway#4055

a11y concerns regarding tables in hourglass were identified within the following templates:

These two pages contain tables which have three headings per data-cell, and they need to be associated with each data cell as per [WC3 Table Tutorial](https://www.w3.org/WAI/tutorials/tables/multi-level/#table-with-three-headers-related-to-each-data-cell)
- [x] components/supporters/demo_bands.html
- [x] components/supporters/demo_table.html

It appears the table on the page below is being used here for a layout - this should be created by styling not by table.
- [ ] templates/custom/governance-and-finances.html
